### PR TITLE
Reactivate Couchbase for SB3

### LINKED ISSUE #27490: Couchbase daily builds are failing
##### **Overview of the issue**

```
Error:  'dependencies.dependency.version' for com.github.differentway:couchmove:jar is missing. @ line 192, column 21
```

https://github.com/hipster-labs/jhipster-daily-builds/actions/runs/11157110942/job/31010784992

##### **Motivation for or Use Case**

Daily builds should pass.

##### **Suggest a Fix**

Add version for couchmove.

##### **JHipster Version(s)**

8.7.1

- [x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by kamilkrzywanski**
Fix for issue #27490 
Single tests might fails yet but most of them should be already is green :) 

Fixes https://github.com/jhipster/generator-jhipster/issues/26577

### CODE REVIEW

**Comment by mshima** [comment]
Hi @kamilkrzywanski you can enable CI by removing:
https://github.com/jhipster/generator-jhipster/blob/62462b98f4a6afb97896f0b11ef282646acb776a/test-integration/workflow-samples/angular.json#L38
And:
https://github.com/jhipster/generator-jhipster/blob/62462b98f4a6afb97896f0b11ef282646acb776a/test-integration/workflow-samples/angular.json#L99

**Comment by kamilkrzywanski** [comment]
@mshima with your sort fix now fails 2 checks instead of 1

File: generators/spring-boot/templates/pom.xml.ejs [hunk 1]
```diff
@@ -527,6 +527,9 @@
                     </executions>
                     <configuration>
                         <mainClass>${start-class}</mainClass>
+<%_ if (databaseTypeCouchbase) { _%>
+                        <loaderImplementation>CLASSIC</loaderImplementation>
```
Comment (mshima) [inline generators/spring-boot/templates/pom.xml.ejs:531]: Should be applied to gradle.

File: generators/spring-boot/templates/src/test/java/_package_/_entityPackage_/web/rest/_entityClass_ResourceIT.java.ejs [hunk 2]
```diff
@@ -1797,7 +1797,7 @@ _%>
         <%= persistInstance %>.set<%= primaryKey.nameCapitalized %>(<%- field.fieldJavaValueGenerator %>);
     <%_ } _%>
   <%_ } _%>
-        inserted<%= persistClass %> = <%= entityInstance %>Repository.<%= saveMethod %>(<%= persistInstance %>)<%= callBlock %>;
+       <%= entityInstance %>Repository.<%= saveMethod %>(<%= persistInstance %>)<%= callBlock %>;
```
Comment (mshima) [inline generators/spring-boot/templates/src/test/java/_package_/_entityPackage_/web/rest/_entityClass_ResourceIT.java.ejs:1800]: Should be reverted or applied to couchbase only

**Review by mshima — COMMENTED** [review]
Snapshots should be updated with `npm run update-snapshots`.

**Comment by kamilkrzywanski** [comment]
@mshima I can't sign agrrement for kkrzywanski. It's required?

**Comment by mshima** [comment]
You can undo last 2 commits. `git reset @~2` and commit with correct user.

**Comment by kamilkrzywanski** [comment]
If you're able to do it feel free, otherwise will do it later.

**Comment by mshima** [comment]
The sign issue can be ignore since we can remove the contributor when squashing, but snapshots should be updated.

**Comment by mshima** [comment]
For some reason running test using prod profile fails:
https://github.com/hipster-labs/jhipster-daily-builds/actions/runs/12018296891/job/33503896516

Any idea?

**Comment by kamilkrzywanski** [comment]
@mshima  idk, will inwestigate on wednesday :) If you have free time then my hint is: when you put sleep before call api then test pass :)  
image

**Comment by kamilkrzywanski** [comment]
@mshima 
https://github.com/spring-projects/spring-data-couchbase/issues/1893
https://github.com/spring-projects/spring-data-couchbase/issues/1797

Looks similar. probably not jhipster implementation issue :)

**Comment by mshima** [comment]
Let’s merge and check the daily builds tomorrow.

**Review by mshima — APPROVED** [review]

**Comment by mshima** [comment]
> @mshima idk, will inwestigate on wednesday :) If you have free time then my hint is: when you put sleep before call api then test pass :) image

@kamilkrzywanski can you add the sleep?

**Comment by kamilkrzywanski** [comment]
@mshima 
Probably I found better solution, but on this weekend i'm bussy, will commit on Sunday/Monday :)

**Comment by mshima** [comment]
Great thanks.

**Comment by DanielFran** [comment]
> $100 bug bounty claim https://opencollective.com/generator-jhipster/expenses/229701

@kamilkrzywanski approved

**Comment by kamilkrzywanski** [comment]
@mshima as told, PR created https://github.com/jhipster/generator-jhipster/pull/28039

**Pull request merged**